lib/rubocop/cop/team.rb

```diff
@@ -160,8 +160,8 @@ def investigate_partial(cops, processed_source, offset:, original:)
       def roundup_relevant_cops(processed_source)
         cops.select do |cop|
           next true if processed_source.comment_config.cop_opted_in?(cop)
-          next false unless @registry.enabled?(cop, @config)
           next false if cop.excluded_file?(processed_source.file_path)
+          next false unless @registry.enabled?(cop, @config)
```

Comment:
This is also the same if the order of the conditions is meaningful.

Reply:
The order is not meaningful, I just swapped them because `cop.excluded_file?` will short circuit more often (in my opinion) and is cheaper to compute.